the_board/lib/pages/workroomPage.dart

```diff
@@ -0,0 +1,378 @@
+
+class WorkroomPage extends StatelessWidget {
+  @override
+  Widget build(BuildContext context) {
+    return MaterialApp(
+      debugShowCheckedModeBanner: false,
+      initialRoute: '/',
+      routes: <String, WidgetBuilder>{
+        '/': (BuildContext context) => WorkRoomList(),
+        '/chat': (BuildContext context) => ChatPage(),
+      },
+      theme: ThemeData(
+        pageTransitionsTheme: const PageTransitionsTheme(
+          builders: <TargetPlatform, PageTransitionsBuilder>{
+            TargetPlatform.android: CupertinoPageTransitionsBuilder(),
+            TargetPlatform.iOS: CupertinoPageTransitionsBuilder(),
+          },
+        ),
+      ),
+    );
+  }
+}
+
+class WorkRoomList extends StatelessWidget {
+  final List workrooms = [
+    [
+      true,
+      "UFM.5",
+      "5.00PM",
+      "Can someone send the link to the Jamboard from the lesson?",
+      [
+        ["Felix Waller", "1:24 PM", "Ah - thanks for the explanation!"],
+        ["Charlie Benello", "1:04 PM", "https://www.cl.cam.ac.uk"]
+      ]
+    ],
+    [
+      true,
+      "UCO.2",
+      "1.34PM",
+      "Can someone send the link to the Jamboard from the lesson?",
+      [
+        ["Felix Waller", "1:24 PM", "Ah - thanks for the explanation!"],
+        ["Felix Waller", "1:24 PM", "Ah - thanks for the explanation!"]
+      ]
+    ],
+    [
+      false,
+      "Upper Eighth",
+      "Yesterday",
+      "Can someone send the link to the Jamboard from the lesson?",
+      [
+        ["Felix Waller", "1:24 PM", "Ah - thanks for the explanation!"],
+        ["Felix Waller", "1:24 PM", "Ah - thanks for the explanation!"]
+      ]
+    ],
+  ];
+
+  Widget workroomTile(List room, BuildContext context) {
+    IconData classIcon;
+    if (room[0]) {
+      classIcon = Icons.book;
+    } else {
+      classIcon = Icons.sports_cricket;
+    }
+
+    return Padding(
+      padding: const EdgeInsets.all(8.0),
+      child: GestureDetector(
+          child: Container(
+            child: Row(
+              children: [
+                Padding(
+                  padding: const EdgeInsets.fromLTRB(0, 0, 8.0, 0),
+                  child: Icon(classIcon),
+                ),
+                Flexible(
+                  child: Column(
+                    crossAxisAlignment: CrossAxisAlignment.start,
+                    children: [
+                      Material(
+                        child: Text(
+                          room[1],
+                          style: TextStyle(
+                              fontWeight: FontWeight.bold), // Room name
+                          textAlign: TextAlign.left,
+                          maxLines: 1,
+                        ),
+                      ),
+                      Material(
+                        child: Text(
+                          room[3],
+                          maxLines: 2,
+                          style:
+                              TextStyle(color: Colors.grey[700]), // Last chat
```

Comment:
Ideally at some point for all this theming, we'll write a themes.dart bit to keep consistency